packages/react/src/themes/teams/components/Input/inputVariables.ts

```diff
@@ -26,7 +26,7 @@ export default (siteVars): InputVariables => ({
   fontSize: siteVars.fontSizes.medium,
 
   iconPosition: 'absolute',
-  iconRight: pxToRem(2),
+  iconRight: pxToRem(10),
```

Comment:
removed the margin of 8px from `Icon` meaning we need to add the difference here